aws/README.md

```diff
@@ -0,0 +1,28 @@
+# AWS EC2
+
+This small setup can be used to provision Flatcar nodes on [AWS EC2](https://aws.amazon.com/ec2/).
+
+Create a `terraform.tfvars` file that lists your preferences. Like this one:
+
+```
+cluster_name            = "mycluster"
+machines                = ["mynode"]
+ssh_keys                = ["ssh-rsa AA... me@mail.net"]
+```
+
+The machine name listed in the `machines` variable is used to retrieve the corresponding [Container Linux Config](https://kinvolk.io/docs/flatcar-container-linux/latest/container-linux-config-transpiler/configuration/). For each machine in the list, you should have a `machine-NAME.yaml.tmpl` file with a corresponding name. An example file `machine-mynode.yaml.tmpl` for `mynode` is already provided. The SSH key used there is not really necessary since we already set it as VM attribute.
```

Comment:
```suggestion
The machine name listed in the `machines` variable is used to retrieve the corresponding [Container Linux Config](https://www.flatcar.org/docs/latest/provisioning/cl-config/). For each machine in the list, you should have a `machine-NAME.yaml.tmpl` file with a corresponding name. An example file `machine-mynode.yaml.tmpl` for `mynode` is already provided. The SSH key used there is not really necessary since we already set it as VM attribute.
```